test-kotlin/src/com/intellij/rt/coverage/kotlin/KotlinCoverageStatusTest.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2000-2020 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intellij.rt.coverage.kotlin
+
+
+import com.intellij.rt.coverage.kotlin.LineStatus.FULL
+import org.junit.After
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Test
+import java.io.File
+
+
+class KotlinCoverageStatusTest {
+    private var myDataFile: File? = null
+
+    @Before
+    fun setUp() {
+        myDataFile = File("data")
```

Comment:
you create file in tests working directory? I would rather create temp files inside temp directory